tests/testthat/_snaps/rmarkdown.md

```diff
@@ -1,23 +1,104 @@
-# render_rmarkdown yields useful error
+# render_rmarkdown copies image files in subdirectories [plain]
+
+    Code
+      render_rmarkdown(pkg, "assets/vignette-with-img.Rmd", "test.html")
+
+# render_rmarkdown copies image files in subdirectories [ansi]
+
+    Code
+      render_rmarkdown(pkg, "assets/vignette-with-img.Rmd", "test.html")
+
+# render_rmarkdown copies image files in subdirectories [unicode]
+
+    Code
+      render_rmarkdown(pkg, "assets/vignette-with-img.Rmd", "test.html")
+
+# render_rmarkdown copies image files in subdirectories [fancy]
+
+    Code
+      render_rmarkdown(pkg, "assets/vignette-with-img.Rmd", "test.html")
+
+# render_rmarkdown yields useful error [plain]
+
+    Code
+      render_rmarkdown(pkg, "assets/pandoc-fail.Rmd", "test.html", output_format = rmarkdown::html_document(
+        pandoc_args = "--fail-if-warnings"))
+    Condition
+      Error in `value[[3L]]()`:
```

Comment:
This error (`` `Error in `value[[3L]]()` ``) is a little ugly and uninformative. It comes out of a `tryCatch` so I think it may be related to the call stack, but not sure how to fix.

Reply:
Probably need to use https://rlang.r-lib.org/reference/topic-error-call.html

Reply:
I don't think it's worth the effort to figure this out, the error is pretty clear.